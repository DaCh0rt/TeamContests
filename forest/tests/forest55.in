948 749
-454 6037
-7210 3538
5003 -135
505 -1048
5970 -2330
-8218 -572
6794 -1852
2100 -8575
6594 3222
9389 -3985
-2547 -8909
-480 -7266
6197 6251
-8124 -84
5904 9160
-5522 -7980
4751 -9336
-8398 -695
-5434 -3503
-3163 -7514
2480 -7373
820 7882
2518 -22
5419 3546
-5040 656
-7941 -2940
3557 -9526
-910 -5686
-1403 2267
3538 2109
-5474 8950
-4677 -2156
-9252 -1606
-4248 -3993
-3701 -5180
-3184 7633
3144 -8178
-9472 8787
9220 5944
-2865 1631
-7973 2156
-4754 1339
-3233 6551
-5499 8279
-2045 6735
-3138 -4882
976 -2523
1455 -9297
-5252 -8825
9247 5237
-8867 9582
306 2274
2203 5356
2561 -9459
-7546 3571
2715 4026
-1227 -5124
5103 -3588
6884 5432
-6857 3424
6627 -5497
1653 9451
-6733 -4058
-8520 -9043
-9500 -3336
-2891 6649
-1595 4144
-4191 5744
-9416 6123
942 -22
-1020 -7820
4044 2721
2125 -7792
-2430 -6329
-5742 8600
8145 -3331
-768 -387
-2490 -7304
7225 884
7292 241
-8414 -2156
-2756 3368
2416 -1146
-8688 9272
-2983 -7521
60 -806
-4379 -5389
4566 7079
6312 -6257
-1569 -907
-6685 -7564
4291 -3945
5097 -5978
-4029 -8629
7088 -2470
9952 -7860
-6784 -1398
-3411 1221
-3139 -5338
7897 -1174
4121 -4637
6213 7658
8206 9833
2738 -254
-9748 -4945
-4725 -6690
-7047 3747
3780 8024
3063 4006
-9364 -8591
-5306 -7583
4269 2704
6658 5430
-8202 7082
-2041 2126
-8780 5627
-7347 6097
-7023 -4164
-5511 -157
-8995 7166
-6061 -1859
1825 -1294
2001 -4702
-5566 -9114
-4157 1309
5627 7691
8423 445
2628 -4073
1289 1778
-7980 -169
696 8988
8080 -2022
7276 8582
-4386 -3660
9846 2024
4772 -7832
7744 37
-6474 -4318
-8742 -1480
312 -5764
-8498 -7623
-5915 4722
5877 7721
8814 1800
-1731 -9081
-8253 1446
-962 6413
-8676 -9130
-8840 -9255
-3467 6084
7494 -8080
-6129 -5717
1176 4996
1395 -6160
-6158 -5224
7842 262
-3541 -7814
2287 -2122
3140 7101
-9629 548
-367 5790
8812 1144
-1187 -9092
-4427 -21
-9024 -3119
-1990 -712
120 -5800
9766 -2019
7896 -5244
3144 -5877
3138 6331
-529 -3898
-499 -5446
-9108 6028
-5495 3815
1717 -9112
963 2855
-4964 -9029
-637 401
9360 -2268
8740 -4270
2591 -3091
-8448 1794
-1210 9318
8480 -5533
-29 4424
-7604 7812
-1070 -1002
-5740 -7511
4529 -5612
-2676 -9623
1544 -6242
305 -6561
-1789 -2910
4469 -2262
-2231 2682
-8090 7479
4690 9876
8319 493
6426 -9418
2883 8373
-514 -8662
-7995 4176
-3234 348
-2428 -3822
-72 -6499
5685 -9009
6961 9952
-5981 3469
3523 9251
4888 4380
7232 -6634
-8340 -9245
-6469 5281
-3019 -6136
3211 -6870
1069 3373
-7761 1167
-3468 -6729
-6687 -4505
-2913 -8923
-1756 -7944
-2725 -6400
3477 -4664
2193 1872
8472 -4199
-8557 2620
-5945 4647
4723 7053
-5534 2258
6800 -7540
7436 7618
-2635 -5641
4613 -9046
-8290 5156
2194 280
3979 -3926
6534 -1418
4627 4081
9396 -1863
-6838 4028
-9689 5279
-3990 693
7978 -9312
7541 4391
5761 7295
-7676 8309
3268 9063
5245 -6646
5986 3152
-3064 8878
1120 3738
1505 -1587
-8974 931
-9719 -7254
-7671 -2202
-2035 -6313
9759 -566
3798 5483
1639 3049
-2600 -2558
-4653 -283
8858 -741
-1289 1139
7851 -7545
-3352 2983
-9905 1561
-4998 9852
6464 7982
-4312 -7702
-5392 -5564
9708 -1333
-960 9910
-7503 7958
3796 8846
-9834 4106
4165 -5098
-8543 5845
3918 -5362
6477 -7957
-7289 -9577
-5933 2847
-7581 -7319
-5055 3997
-7572 4028
-3981 -4379
525 -4783
-3277 7993
8141 -4371
7860 -2375
-6291 -3851
-413 6774
-1027 8719
-5213 -6690
-4181 8953
856 2797
9590 -6212
9493 -9674
-8427 -8452
8008 8969
2882 -6068
-1330 9258
4854 5123
2406 -8483
-7759 4491
6792 -6044
-2171 4755
-1870 1487
2702 -1010
-5498 -8215
7986 -4856
-657 -2708
277 -5380
4102 3473
4804 -2366
-3835 -7019
-9382 3205
-892 2210
-8628 -9700
1538 -8201
4175 6798
-7288 -7021
9815 8986
-7619 6000
-5955 -6817
8484 -726
-2291 -6092
-3526 -5441
-4467 6960
5894 -1200
8009 1306
6293 1872
1667 -3548
2846 -6594
-4503 432
-4717 -2328
4031 3737
4667 7637
6265 -8280
9918 68
3249 -6491
-4009 1963
-5594 9245
-6311 -1886
-4256 -7293
3860 2718
7663 4995
4452 1281
-9061 6963
6308 7167
6037 2340
-9231 -3935
-5841 403
9662 -7125
3787 9969
-7261 430
2727 -8794
7094 6238
2540 9185
-8781 -7515
2361 6350
5291 -1024
-4225 -4536
5866 -8965
9533 2433
-1610 2572
-1832 -5573
-1754 3825
6570 4318
261 -8813
7046 -8543
-9287 1022
9118 4940
-4323 4599
8109 -2606
3295 1879
2735 -3934
9321 4388
2253 7366
497 2016
7423 -9619
1934 7770
9801 6239
3149 -476
2777 2164
-5169 -7338
-8983 -9136
-436 -9854
-5785 7833
5141 -6922
-332 8492
5755 -6395
-4981 9476
-7774 7178
-3716 -8230
-6362 -7712
1333 -8066
5977 -2711
6088 -6480
-1181 5834
-824 -114
5465 -4014
-8526 -8279
-8491 -8290
1796 -5649
-6443 -3621
5444 -5211
-7734 -5270
2843 -3129
-2381 -9414
9777 -3852
-8750 9189
4369 -708
-6660 -1519
6199 3670
-1322 7714
3884 5373
-406 -9455
-261 365
-9799 8115
-841 86
7894 -4307
-6616 -8851
2094 2430
1207 -3034
1806 -6347
5128 9063
8400 7138
-2566 1503
-4036 -7588
-6500 -4824
-7249 6809
2933 1609
5934 5049
5471 -1835
-1349 4339
-8879 -981
7644 -6196
5022 -7378
2350 8468
3923 -3433
3705 -2021
1408 5996
-1850 2631
-619 8255
-2883 -2884
8581 -9439
3714 3036
2957 -2183
2788 -8163
-8384 -9948
-6254 7544
-5623 -805
6418 558
-5929 8822
5224 3849
1104 3698
8741 -4650
-5658 -6779
9856 -8995
9225 5234
-1289 -3356
-3743 -1215
8531 1752
-7202 2880
-1375 1276
-2196 -3908
4829 -3464
-515 9749
9552 -3017
3525 2338
5467 7157
-7539 9807
7547 -9377
-1036 1316
4778 7294
6866 -4341
-5004 -9080
9944 3314
-4361 7438
7552 2669
-8646 684
-7682 -9618
-1289 -5460
9127 9863
84 -9642
-3111 5182
2390 -2218
-4889 3847
-3339 -3193
7377 6629
-4673 -3752
8278 -8662
-9593 3831
2541 9117
9249 8965
4707 -5431
-8370 4054
5536 -7301
-2576 323
-8285 -2757
5109 -6681
-3123 -8777
-9087 -5702
-3376 2114
8635 -1202
-1698 5507
1428 8863
-1396 3819
7768 -2610
5417 7981
-5098 -8646
-1061 -8822
4733 1636
-3634 -5141
7922 -5560
-6468 -5426
6525 -3846
-6641 3054
-6669 -9065
-4579 -5406
-8232 3800
-8244 1414
-6228 -2680
1657 -401
5760 6279
1713 7176
9536 3853
-7415 9834
8853 8188
-4556 2676
7146 -7326
-629 4457
7605 1079
7326 -5124
5856 5371
4264 -6273
-4464 -5790
-597 5779
2988 3413
-5009 -3905
9158 -2607
-7053 4629
2573 -2250
8771 -4476
5792 -6745
-6979 4456
-9622 1931
5238 6013
1984 -7456
2832 -6707
-1334 -3146
-9881 -1881
1325 -3083
3080 9025
-6106 7884
-2602 -8733
4563 2957
5741 5682
9943 2300
-1855 3208
3392 -3492
-7602 -8924
5467 1281
-8923 1146
2158 -3357
-7144 9651
3661 7794
-6955 2766
4254 2143
8978 -4748
1737 3198
-5648 -2531
5585 3720
-8438 -9663
-4020 -8470
-4524 3763
2526 -8582
596 5691
7277 5865
-8364 6036
1834 -3300
583 -4990
-4448 -7955
-9246 3652
7108 3461
1707 6982
9271 -5953
1098 9102
-9181 -2264
-879 4343
-4527 -7502
9666 -9350
6699 152
-9674 9761
-4501 -8234
5821 -4577
-4001 6228
8345 -3667
-1690 -6211
-4333 9693
5914 3717
9590 -3321
38 8691
-9697 5737
-9621 -3735
-9708 -4340
6902 -9534
-5021 -2160
4146 5393
9969 -7025
4383 8720
-4468 -6859
-6935 -6456
8812 -1837
-7643 -4839
9222 1796
7097 5630
2452 -9746
7864 8992
-2953 6618
-2110 1018
-101 6759
-9735 -1517
-1968 -87
-7884 55
9111 -5350
-5251 -1029
-5398 -2375
-3410 6224
-4520 -6735
913 8885
8755 -2125
-2524 -5552
-7429 -1485
6156 -6995
-8291 -4565
-5999 -9985
7041 -5830
-7798 -2806
1402 7485
6949 -3511
-3940 -6386
4280 9352
4293 9860
8488 2471
9772 -1076
4933 -1452
-150 -4120
5617 -3352
-4468 -4889
-6628 1798
-6042 -2761
6704 -8464
5898 -2046
9476 7679
-8013 6995
-5667 4888
-9172 9443
-8849 -7471
5218 3264
-2595 2132
9810 -1172
-5454 615
3548 -5406
-9000 8244
-6590 -1136
679 5795
8825 2780
1455 -3701
-6971 1410
-1128 -3763
992 5928
-2874 -8457
7122 4960
6985 9292
2766 -3071
3940 -6336
2384 -1786
-9850 7250
-9555 1087
-3579 6489
-3955 662
-5137 8386
4041 6606
-4649 -8006
-4052 4651
4110 -7163
5247 -6971
1173 2845
-196 -1750
-8524 7047
2676 -4923
5037 -3764
4825 1805
4672 -5758
5699 902
2064 7932
-7718 6956
-5401 4894
8408 -4032
8077 -6364
7968 6629
3466 4702
-9742 -6121
1221 3299
-8859 -4849
-4467 7644
-6729 6424
-5639 -3751
-8385 -9747
260 -4791
9337 -9584
-8611 7444
6654 -9866
-6936 6715
76 6518
6475 6729
9885 3932
5468 -82
1976 3423
3452 6060
4623 -2869
-9740 -4598
-1457 130
8016 -9624
8014 6002
3328 8286
-3614 -2591
263 7070
-6156 1840
-957 -6433
-6389 -3653
-210 2727
-6095 -9287
-1561 -5880
3502 -9089
9927 8577
8710 -5907
1980 706
-6984 -9290
-9867 9927
-5071 5548
-4562 9351
6430 -8189
6432 4768
3526 5643
9952 2968
7120 -4802
-3658 -2885
288 9738
-9874 -839
8537 -9856
-5756 7027
3840 8993
5937 8952
5503 4033
6555 -1782
5995 -5359
9134 644
-8897 3673
5258 -5798
-8041 -2943
2529 -7906
-4225 -5729
2055 585
-8247 8039
-533 -2028
-7964 697
-868 5490
-1437 -7031
-550 -5930
-2047 -3742
-7947 6682
-6854 -9369
-6950 9991
-6483 -6307
-5464 7012
-939 3149
-9778 2562
-3747 -9584
-1447 7619
6343 113
5041 4474
-422 8757
8907 5169
4395 -8856
-6090 -3324
-2235 -4444
-761 -9697
-6932 -6073
-6511 4285
-5849 -7511
283 6143
-9174 -9477
4608 8361
7876 -1897
-6797 8906
-9063 -2934
2436 -3809
-9901 -203
841 8572
6273 -6502
-1945 836
5035 4405
6750 -8802
1654 -6915
9726 -6271
-8684 -3190
8991 -9854
-7518 9651
-6974 7792
-7835 5118
1135 -5085
-9143 1830
-8475 426
-1244 9070
-4858 -103
6709 -739
5591 -7989
4486 -67
9111 1487
-5197 3974
1534 -4460
1093 658
7432 -7384
-6803 8865
-8598 5994
-6379 -190
2081 6965
9738 -4097
-7862 -4347
6708 -3093
-5096 -5192
9221 -9677
-1294 2240
2135 9521
8081 -1138
-4081 -1718
7867 7313
1164 3322
4635 -5047
435 -8634
-7928 -702
-5942 -5139
-69 1077
-2131 -4413
-9396 9200
-3062 4851
-8180 -8906
-4985 7051
6817 1903
-9266 -652
6305 6745
1082 -5164
-4981 2259
-874 -8577
2531 9386
-60 129
-4933 -3795
-5447 4451
9443 -1054
9302 360
8057 8685
-882 2964
2186 -7092
5068 -6660
6009 5503
-2265 -2578
1323 9800
-7915 7930
-8438 -3405
-5889 -675
3326 -2992
3882 -2406
-5466 -4222
5561 8421
-1551 8020
1537 -5140
-7807 -7527
-8599 -4521
8778 -2755
-4672 8083
6300 9147
-8771 8966
8503 7103
-8526 4946
-3205 6143
-1599 3507
-3818 6361
-8852 5397
3321 7525
1196 4415
-4883 1732
-8134 2841
134 8769
-5874 4639
-2466 9153
-5148 3079
-9925 -6369
-8182 -1448
640 -8444
546 -2290
-9480 -3958
-222 2943
-2555 -3656
-7727 85
8788 4838
-7004 9920
-105 -3074
5712 5920
4850 -9521
4074 -4819
-1455 -3135
-6663 2449
-7493 4454
222 7027
-2023 8591
-1443 -5242
-3779 1404
-9026 -348
7248 -6663
9009 -3883
-9561 -5667
-4892 -9710
2691 1924
557 5645
8915 4223
2897 -7843
3753 -2690
-4630 136
-2290 -2984
4909 7123
4274 9850
-6356 -2616
7955 -522
-104 -4853
3200 -8139
-6501 5955
7854 -9289
689 -9557
-4754 1753
218 -6728
67 -2300
-2835 -1520
-4945 8551
327 2823
-4640 -1287
