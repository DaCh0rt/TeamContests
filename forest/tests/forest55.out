81910.17 451939982.50
